metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -94,6 +95,31 @@
          vals
          first))))
 
+(defn- append-stmt
```

Comment:
Can we do a GTM to review these database query changes? I'm wondering why these changes are needed, and not sure about doing string manipulation on SQL.

Reply:
These are done for performance optimization. The existing code for finding concepts with params are not efficient for large volume concepts such as variables. We should consolidate the params search code with this function where the filtering of latest revision of concept is done through sql rather than code, and is much more efficient. I don't have time to complete the refactoring within this PR. We will do it in a separate issue later.